package.json

```diff
@@ -1,15 +1,17 @@
 {
   "name": "fixed-data-table-2",
-  "version": "0.8.28",
+  "version": "1.0.0-beta.30",
```

Comment:
Do we want to switch this to not be beta anymore?

1.0.1?

Reply:
sounds good.

Btw why not `v1.0.0` instead of `v1.0.1` ?

Reply:
I was thinking of the betas as v1.0.0.  But looking at react they go from 15.4.0-rc.4 to 15.4.0, so I'm good w/ the release being 1.0.0.